src/ReportPortal.Shared/Extensibility/Embedded/Analytics/AnalyticsReportEventsObserver.cs

```diff
@@ -103,6 +99,32 @@ public void Initialize(IReportEventsSource reportEventsSource)
 
         private Task _sendGaUsageTask;
 
+        static HttpClient GetHttpClient(IConfiguration configuration)
+        {
+            if (_httpClient != null) 
+                return _httpClient;
+
+            lock (_httpClientLock)
+            {
+                if (_httpClient != null) 
+                    return _httpClient;
+                
+                var handler = new HttpClientHandler();
+#if !NET462
```

Comment:
else? For net462 we can use
```csharp
ServicePointManager.ServerCertificateValidationCallback += (sender, cert, chain, sslPolicyErrors) => true;
```

Reply:
This was effectively just a refactor of the existing code (which didn't handle that case), but happy to make it better while we're here 👍